### PR TITLE
Fix #1672: Prevent creating activation scoped temporary key pair for FIDO2 activations

### LINKED ISSUE #1672: Prevent creating activation scoped temporary key pair for FIDO2 activations
While creating a temporary encryption key pair for FIDO2 activations does not hurt anything, there is nothing such a key could be used...

### PR DESCRIPTION
**Pull request by petrdvorak**

### CODE REVIEW

File: powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/TemporaryKeyBehavior.java [hunk 1]
```diff
@@ -278,21 +279,23 @@ private TemporaryKeyResult obtainTemporaryKeyResult(TemporaryPublicKeyRequestCla
                 return result;
             } else {
 
-                final String appId = applicationVersionEntity.getApplication().getId();
+                final Long appId = applicationVersionEntity.getApplication().getRid();
```
Comment (banterCZ) [inline powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/TemporaryKeyBehavior.java:282]: 🤔 Is this change needed (the `id` should be also unique)? If so, I suggest renaming the variable, because it is confusing now (`id` vs `rid`).

**Review by banterCZ — APPROVED** [review]
The branch for FIDO2 ActivationProtocol looks good.

**Review by romanstrobl — APPROVED** [review]
Approved.

File: powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/TemporaryKeyBehavior.java (hunk 1 above)
Comment (petrdvorak) [reply powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/TemporaryKeyBehavior.java:282]: We can refactor it... I used `rid` here because the comparison could be slightly faster...
